### PR TITLE
Add Groestlcoin device tests

### PR DESCRIPTION
**Pull request by yura-pakhuchiy**
Bitcoin tests do not fully cover Groestlcoin because GRS uses some hashes differently from Bitcoin and changes to other coins may accidentally break GRS. Based on GRS tests in T2 repo.

### CODE REVIEW

File: trezorlib/tests/device_tests/test_msg_getaddress.py [hunk 1]
```diff
@@ -101,6 +101,60 @@ def test_bch(self):
             == "bitcoincash:qzc5q87w069lzg7g3gzx0c8dz83mn7l02scej5aluw"
         )
 
+    def test_grs(self):
+        self.setup_mnemonic_allallall()
+        assert (
+            btc.get_address(
+                self.client,
+                "Groestlcoin",
+                [44 | 0x80000000, 17 | 0x80000000, 0 | 0x80000000, 0, 0],
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_msg_getaddress.py]: please use `parse_path`

File: trezorlib/tests/device_tests/test_msg_getaddress.py [hunk 2]
```diff
@@ -101,6 +101,60 @@ def test_bch(self):
...
+        assert (
+            btc.get_address(
+                self.client,
+                "Groestlcoin",
+                [44 | 0x80000000, 17 | 0x80000000, 0 | 0x80000000, 1, 1],
+            )
+            == "Fmhtxeh7YdCBkyQF7AQG4QnY8y3rJg89di"
+        )
+
+    def test_tgrs(self):
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_msg_getaddress.py]: we probably don't need to test tgrs separately

File: trezorlib/tests/device_tests/test_msg_signtx_grs.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+from .test_msg_signtx import check_sign_tx
+
+
+class TestMsgSigntxGRS(TrezorTest):
+    def test_legacy(self):
+        # http://blockbook.groestlcoin.org/tx/f56521b17b828897f72b30dd21b0192fd942342e89acbb06abf1d446282c30f5
+        self.setup_mnemonic_allallall()
+
+        inp1 = proto.TxInputType(
+            address_n=[
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_msg_signtx_grs.py]: please either use `parse_path` or the `H_()` shortcut from `trezorlib.tools`

**Comment by yura-pakhuchiy** [comment]
@matejcik Thank you for the review! I've pushed update.

**Comment by matejcik** [comment]
looking good.
just to make sure, do these tests pass with current T1 and TT master?

**Comment by yura-pakhuchiy** [comment]
I've tested against trezor-core 8be4ed41bc7f3ad841166a731817b68f302d8a50 and mcu 964a622bb512aa85cfcc3e451fc70729cc15bb4f. They pass unless few more recent commits broke something.

**Comment by matejcik** [comment]
alright, merged, thanks :+1:

**Pull request merged**